src/commands/deployUtils/versions.js

```diff
@@ -59,6 +60,12 @@ module.exports = (config, { operations, allocations }) => {
 				id,
 				deployment: { container: { image } },
 				envVariables,
+				automaticScaling: Object.assign(
+					baseConfig.automaticScaling || {},
+					{
+						minTotalInstances: minInstances,
```

Comment:
Is there a reason for the slightly different names?

Reply:
I know app.json uses minTotalInstances but not sure feels like it might be slightly confusing... But could not a big deal

Reply:
yeah I'm ambivalent as well - since we're managing multiple deployments that each have a minimum number of instances, i kinda feel like calling the option `minTotalInstances` would be misleading since there will be more instances running in other deployments... I think just calling this `minInstances` is intuitive enough without it being toooo misleading